sympy/integrals/rubi/utility_function.py

```diff
@@ -6062,7 +6062,7 @@ def SimpHelp(u, x):
                 w = i + w
         v = SmartSimplify(v)
         if SumQ(w):
-            w = Add(*[SimpHelp(i, x) for i in w])
+            w = Add(*[SimpHelp(i, x) for i in w.args])
```

Comment:
Why this change?

Reply:
I think that is an error, because in the above part of code , we are doing `w = i + w`. So 
`for i in w :` will give error as 
```
w = Add(*[SimpHelp(i, x) for i in w])
TypeError: 'Add' object is not iterable
```